src/sentry/static/sentry/app/stores/guideStore.tsx

```diff
@@ -90,68 +97,80 @@ const GuideStore = Reflux.createStore({
     this.trigger(this.state);
   },
 
-  onRegisterAnchor(anchor) {
-    this.state.anchors.add(anchor);
+  onRegisterAnchor(target: string) {
+    this.state.anchors.add(target);
     this.updateCurrentGuide();
   },
 
-  onUnregisterAnchor(anchor) {
-    this.state.anchors.delete(anchor);
+  onUnregisterAnchor(target: string) {
+    this.state.anchors.delete(target);
     this.updateCurrentGuide();
   },
 
-  recordCue(id) {
+  recordCue(id: number) {
     const data = {
       eventKey: 'assistant.guide_cued',
       eventName: 'Assistant Guide Cued',
       guide: id,
+      organization_id: this.state.orgId,
     };
-    if (this.state.org) {
-      data.organization_id = this.state.org.id;
-    }
     trackAnalyticsEvent(data);
   },
 
-  updatePrevGuide(bestGuide) {
+  updateGuidesWithContent(data: GuideData) {
+    try {
+      const content = getGuideContent();
```

Comment:
Can we just pull this up to the module level, since we don't relaly need to call it all the time right now